docs/usage/control-plane-endpoints-and-ports.md

```diff
@@ -0,0 +1,46 @@
+# Endpoints and Ports of a Shoot Control-Plane
+
+With the reversed VPN tunnel, there are no endpoints with open ports in the shoot cluster required by gardener.
+In order to allow communication to the shoots control-plane in the seed cluster, there are shared endpoints for all shoots of a seed cluster.
```

Comment:
In most cases, this might be true, but it is not true if exposure classes are used. Maybe we want to be more explicit. However, I am a bit undecided on this one.

Reply:
On second thought: With multi-zonal seed clusters this is even less true.

Reply:
I wanted to keep it simple here. But you're right, oversimplification can make it wrong.